frontend/tests/components/admin/droplet-client.test.tsx

```
@@ -1,13 +1,33 @@
 import { DropletClient } from "@/components/admin/droplets/droplet-client";
 import { DropletStatus, DropletType, FocusArea, Tag } from "@/types";
-import { render, screen, fireEvent, act } from "@testing-library/react";
+import {
+  render,
+  screen,
+  fireEvent,
+  act,
+  waitFor,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
```

ESLint

[error] 'userEvent' is defined but never used.

(@typescript-eslint/no-unused-vars)